edgraph/access_ee_test.go

```diff
@@ -0,0 +1,180 @@
+package edgraph
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/dgraph-io/dgraph/ee/acl"
+	"github.com/dgraph-io/dgraph/worker"
+	jwt "github.com/dgrijalva/jwt-go"
+)
+
+var (
+	userDataList = []userData{
+		userData{1234567890, "user1", []string{"701", "702"}},
+		userData{2345678901, "user2", []string{"703", "701"}},
+		userData{3456789012, "user3", []string{"702", "703"}},
+	}
+)
+
+func generateJWT(namespace uint64, userId string, groupIds []string, exp int64) (string, error) {
+	var e1 int64
+	var token *jwt.Token
+
+	if exp == 0 {
+		e1 = time.Now().Add(time.Minute * 30).Unix()
+	} else {
+		e1 = exp
+	}
+
+	if groupIds != nil {
```

Comment:
This block could be easier to read by creating the token outside the if branch, then applying the "groups" member value iif `groupsIds != nil`